setup.py

```diff
@@ -31,7 +32,7 @@
         p = subprocess.Popen(["git", "rev-parse", "--short", "HEAD"], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
         out, err = p.communicate()
         if out:
-            version += "+g" + out.decode("utf-8").strip()
+            version += f"+g{out.decode('utf-8').strip()}"
```

Comment:
why was this changed?

Reply:
I think my IDE ran Flynt on the file by mistake, though it would still be good to move concatenated strings to f-strings where appropriate anyway (as a separate PR though - I'll revert this change here).